templates/zerver/help/add-a-bot-or-integration.md

```diff
@@ -0,0 +1,41 @@
+# Add a Bot or Integration
+
+Bots and integrations are features that are accesible to all members of an organization.
+
+## To Add a Bot
+
+Bots can be created from the settings page:
+
+{!go-to-the.md!} [Your Bots](/#settings/your-bots) {!settings.md!}
```

Comment:
`{!go-to-the.md!} [Your Bots](/#settings/your-bots)` is on one line `{!settings.md!}` needs to be on a separate line.